L/Libxls/build_tarballs.jl

```diff
@@ -1,12 +1,14 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using BinaryBuilder
+using BinaryBuilder, Pkg
 name = "libxls"
```

Comment:
I don't know how this ever worked, but this name should match exactly the directory name, case included, and since the package is already registered as "libxsl" the directory should be renamed accordingly.

Reply:
I don't think anyone is using this at the moment (there are no packages in the general registry that depend on this), so I'm wondering whether we could instead adopt `Libxls` for this? And rename the package in the registry to `Libxls_jll` as well?

Reply:
Meh, you can't rename a package in the registry, you have to effectively duplicate it.  I don't think it's worth the effort.

Reply:
I'm trying the `Libxls` spelling now, just to see whether it compiles with that.

Reply:
BTW, we can't have two packages in the registry with the same name modulo the case, so using `Libxls` is very much ***not*** an option.

Reply:
Bump